app/src/pages/Editor/ExportDocumentDialog.tsx

```diff
@@ -41,24 +42,50 @@ export function ExportDocumentDialog(): JSX.Element {
     });
   });
 
+  const popupState = useSelector((state: RootState) => state.editor.present?.exportPopup);
+
+  const firstSelectionWordsString = useSelector((state: RootState) => {
+    const editorState = state.editor.present;
+    if (!editorState) return '';
```

Comment:
This should either early-return if `popupState != 'selection'` or use the first words of the document. Right now it uses the first word of the selection even if the entire document is exported